packages/react-ui/components/Input/Input.styles.ts

```diff
@@ -210,6 +210,7 @@ export const styles = memoizeStyle({
       background: ${t.inputDisabledBg};
       border-color: ${t.inputDisabledBorderColor};
       box-shadow: none;
+      background-clip: padding-box;
```

Comment:
На самом деле `background-clip: padding-box` уже применяется в `.root`, и я бы сказал, что там ему более подходящее место. Но он перетирается в `.disabled` общим свойством `background: ...`.  Думаю, лучше будет поменять background на background-color. Тогда новый background-clip не понадобится.

```suggestion
      background-color: ${t.inputDisabledBg};
      border-color: ${t.inputDisabledBorderColor};
      box-shadow: none;
```